signer/migrations/0003__create_tables.sql

```diff
@@ -46,27 +46,28 @@ CREATE TABLE sbtc_signer.deposit_signers (
     FOREIGN KEY (txid, output_index) REFERENCES sbtc_signer.deposit_requests(txid, output_index) ON DELETE CASCADE
 );
 
-CREATE TABLE sbtc_signer.withdraw_requests (
+CREATE TABLE sbtc_signer.withdrawal_requests (
     request_id BIGINT NOT NULL,
     txid BYTEA NOT NULL,
     block_hash BYTEA NOT NULL,
-    recipient TEXT NOT NULL,
+    recipient BYTEA NOT NULL,
     amount BIGINT NOT NULL,
     max_fee BIGINT NOT NULL,
     sender_address TEXT NOT NULL,
     created_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP NOT NULL,
-    PRIMARY KEY (request_id, block_hash),
+    PRIMARY KEY (request_id, txid, block_hash),
```

Comment:
Just checking -- this is valid because of a reorg situation right? That the same request_id could potentially exist within multiple blocks? But could it really exist within multiple txids within the same block (as a withdrawal in this case)?

Reply:
Actually, I'm pretty sure that it's unnecessary to include the `txid` here. As in a request should be uniquely identified by the `request_id` and the `block_hash`. So I'm pretty sure that it can be safely reverted back to what it was.

Reply:
> That the same request_id could potentially exist within multiple blocks?

Yes that is correct, a `request_id` could exist within multiple blocks because of reorgs.

Reply:
Yeah I'd revert it if it's invalid to have multiple in the same block since we lose the unique constraint this way

Reply:
I'm not sure I follow, I was saying that `(request_id, block_hash)` is probably unique. If that is unique then `(request_id, txid, block_hash)` is necessarily unique as well.

But yeah I'm kinda okay with reverting, with the reason being that it is probably unnecessary. The main reason why I changed the primary key here was to be consistent with how we identify withdrawal requests elsewhere (like in the receipt that is stored on the bitcoin blockchain). It's easier (mentally) for someone new (or me after a month or so) to understand things if we use the same identifier everywhere for the same concept. Not sure how to balance it all, I'll default to what you think here.

Reply:
On phone but I just meant that if
```
request_id, block hash, txid
1                  abc123        1
1                  abc123        2
```
Isn't valid then the PK (or a separate UK) should be on the request id and block, otherwise it opens for multiple txids for the same request and block.

Reply:
Ahh gotcha, makes sense.

Reply:
https://github.com/stacks-network/sbtc/pull/528/commits/fc05924c824bbbed5c6028c06777c683532b0485